WindowsFormsApplication11/Program.cs

```diff
@@ -24,5 +24,13 @@ static void Main()
             Console.WriteLine("Ivan");
             Console.WriteLine("Mykhailov");
         }
+        static void function()
+        {
+            for (int i = 0; i < 10; i++)
+            {
+                Console.WriteLine("Own Branch changes");
```

Comment:
Change code to calculate first  sequence's members

Reply:
Please Accept